project/Newevaluation.html

```diff
@@ -6,85 +6,118 @@
   <style>
     body{
       background-color: rgb(82, 86, 89);
-      margin-top: 50px;
+      margin-top: 3%;
```

Comment:
Nu zic musai ca-i gresit. Problema cu procentaje, cel putin pentru `margin` / `padding` e ca sunt relative la dimensiunea ecranului. Daca ai la dispozitie `1200px` pentru elementul tau, un `margin: 3%` inseamna `36px`, pe cand daca ai doar `12px`, pentru `margin` iti ramane mai putin de `1px`. Depinde de ce anume vrei sa faci, pana la urma, dar e un lucru de avut in vedere. ;)